### PR TITLE
feat: add integration on assignment page

### PR DESCRIPTION
**Pull request by uliviaembun**
## Story/Task

> [Integrasi Assignment Page](https://trello.com/c/y9ntlhVu)

## Summary

> Add integration on all of assignment page

## Screenshot

> 1. Tampilan Daily Quest
> Screenshot from 2023-08-08 21-38-02

> 2. Tampilan Side Quest
> Screenshot from 2023-08-08 21-38-07

> 3. Tampilan Belum Terkumpul
> Screenshot from 2023-08-08 21-20-14

> 4. Tampilan setelah pilih file
> Screenshot from 2023-08-08 21-20-47

> 5. Tampilan Pop Up
> Screenshot from 2023-08-08 21-21-01

> 6. Tampilan saat Tugas Berhasil Dikirim
> Screenshot from 2023-08-08 21-21-11

> 7. Tampilan saat Tugas Terkumpul
> Screenshot from 2023-08-08 21-21-22

> 8. Tampilan saat mau submit tanpa file
> Screenshot from 2023-08-08 21-20-28

> 9. Tampilan Terlambat
> Screenshot from 2023-08-08 21-19-56

### CODE REVIEW

File: src/components/assignment/SubmitPopUp.tsx [hunk 1]
```diff
@@ -1,13 +1,75 @@
...
+        duration: 2000,
+        isClosable: true,
+        position: "top"
+      });
+      submittingFile(false);
+      return;
+    }
+
+    const sanitizedFileName = sanitizeURL("https://cdn.oskmitb.com/"+fileToSend.name);
+    console.log(sanitizedFileName)
```
Comment (pandora-1) [inline src/components/assignment/SubmitPopUp.tsx]: Ini dihapus yaa

File: src/components/assignment/SubmitPopUp.tsx [hunk 2]
```diff
@@ -1,13 +1,75 @@
...
+    await uploadFile(sanitizedFileName, fileToSend, (event) => {
+      if (!event.total) return;
+    });
+
+    try {
+      const result = await uploadMutation.mutateAsync({
+        assignmentId: taskId,
+        filePath: sanitizedFileName
+      });
+      toast({
```
Comment (pandora-1) [inline src/components/assignment/SubmitPopUp.tsx]: Perlu dilakukan pengecekan response dari result tuh 200 apa enggak, bisa jadi dia gagal gitu

**Review by pandora-1 — CHANGES_REQUESTED** [review]

File: src/components/assignment/SubmitPopUp.tsx (hunk 1 above)
Comment (uliviaembun) [reply src/components/assignment/SubmitPopUp.tsx]: console.lognya ya kak

Comment (pandora-1) [reply src/components/assignment/SubmitPopUp.tsx]: iyaa

**Review by pandora-1 — APPROVED** [review]

**Pull request merged**